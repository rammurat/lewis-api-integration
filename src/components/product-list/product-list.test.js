// @ts-nocheck
import React from 'react'
import renderer from 'react-test-renderer';
import { Provider } from 'react-redux'
import List from './product-list'

import Enzyme, { mount } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
Enzyme.configure({ adapter: new Adapter() });

const store = {
    subscribe: () => { },
    dispatch: () => { },
    getState: () => ({
        state: {
            psp: {
                list: [],
                categoryTitle: '',
                totalProducts: 0,
                initialConfig: {
                },
                isNoResult: false,
                errMsg: '',
                isLoading: false
            }
        }
    })
}

it('renders correctly', () => {
    const tree = renderer
        .create(<Provider store={store}><List /></Provider>)
        .toJSON();
    expect(tree).toMatchSnapshot();
});
