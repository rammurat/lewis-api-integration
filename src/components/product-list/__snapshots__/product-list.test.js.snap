// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="psp-page-cnt"
>
  <div
    className="psp-page-cnt__heading"
  />
  <section
    className="psp-page-cnt__list"
  />
</div>
`;
